plugin/src/generators/sentry/generator.ts

```diff
@@ -0,0 +1,21 @@
+import { formatFiles, Tree } from '@nx/devkit';
+import { SentryGeneratorSchema } from './schema';
+import { isExpoApp, isNextApp } from '../../shared/utils';
+import { generateSentryNext, generateSentryExpo } from './utils';
+
+export async function sentryGenerator(
+  tree: Tree,
+  options: SentryGeneratorSchema,
+) {
+  const projectRoot = `apps/${options.directory}`;
```

Comment:
can't we use [selectProject](https://github.com/RonasIT/nx-generators/blob/main/plugin/src/shared/utils/cli-utils.ts#L98) from utils? So we can remove `x-prompt` for directory.

Example: https://github.com/RonasIT/nx-generators/blob/main/plugin/src/generators/react-lib/generator.ts#L22